templates/admin/auth/list.tmpl

```diff
@@ -31,6 +31,7 @@
 							<td><a href="{{AppSubUrl}}/-/admin/auths/{{.ID}}">{{svg "octicon-pencil"}}</a></td>
 						</tr>
 					{{end}}
+					{{template "shared/no_results_found" (dict "rst" .Sources "colspan" 7)}}
```

Comment:
What is R! S! T!

* `reStructuredText (RST, ReST, or reST)`
* `RST Moto https://www.rst-moto.com`
* `Intel® Rapid Storage Technology (Intel® RST) Driver`
* `Rail Staff Travel (RST)`